pom.xml

```diff
@@ -474,12 +428,6 @@
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>org.javassist</groupId>
```

Comment:
Sonar doesn't seem to care about the lack of `javassist`, so I'm going to leave this out for now.